server.go

```diff
@@ -70,6 +71,13 @@ func printLaunchBar(addr, baseURL string, isTLS bool) {
 	log.Println("# └" + bar + "┘")
 }
 
+// check if the directory is accessible
+func checkDirectory(path string) {
+	if _, err := os.Stat(path); os.IsNotExist(err) {
```

Comment:
Let's add this to `splitDirMapping` function.

Reply:
added.